docs/integrations/sources/zoho-crm.md

```diff
@@ -0,0 +1,137 @@
+# Zoho CRM
+
+## Sync overview
+
+The Zoho CRM source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+
+Airbyte uses [REST API](https://www.zoho.com/crm/developer/docs/api/v2/modules-api.html) to fetch data from Zoho CRM.
+
+This Source Connector is based on an [Airbyte CDK](https://docs.airbyte.io/connector-development/cdk-python).
+
+### Output schema
+
+This Source is capable of syncing:
+
+* standard modules available in Zoho CRM account
+* custom modules manually added by user, available in Zoho CRM account
+* custom fields in both standard and custom modules, available in Zoho CRM account
+
+The discovering of Zoho CRM module schema is made dynamically based on Metadata API and should generally take no longer than 10 to 30 seconds.
+
+### Notes:
+
+Some of Zoho CRM Modules may not be available for sync due to limitations of Zoho CRM Edition or permissions scope. For details refer to the [Scopes](https://www.zoho.com/crm/developer/docs/api/v2/scopes.html) section in the Zoho CRM documentation.
+
+Connector streams and schemas are built dynamically on top of Metadata that is available at the same REST API - please see [Modules API](https://www.zoho.com/crm/developer/docs/api/v2/modules-api.html), [Modules Metadata API](https://www.zoho.com/crm/developer/docs/api/v2/module-meta.html), [Fields Metadata API](https://www.zoho.com/crm/developer/docs/api/v2/field-meta.html).
+Therefore, the list of available streams is equivalent to the list of Moules as long as Module Metadata is available for each of them (it describes a set of fields a module consists of) and Fields Metadata is available for each of the fields (it describes each and every field, so we know how to transform it into Airbyte data type). 
```

Comment:
```suggestion
The list of available streams is the list of Modules as long as Module Metadata is available for each of them from the Zoho CRM API, and Fields Metadata is available for each of the fields. If a module you want to sync is not available from this connector, it's because the Zoho CRM API does not make it available. 

```

Reply:
fixed